File: backend/router/es_task.go
package router

// ES 任务 路由
func (this *WebServer) runEsTask() {

	const AbsolutePath = "/api/es_task"
	group := this.engine.Group("ES异步任务", AbsolutePath)
	{
		group.POST(false, "Es任务列表", "/ListAction", this.esTaskController.ListAction)
		group.POST(true, "取消ES任务", "/CancelAction", this.esTaskController.CancelAction)
	}
}

File: frontend/src/api/es-task.ts
import {request} from "@elasticview/plugin-sdk";

const api = '/api/es_task/'

export function ListAction(data) {
  return request({
    url: api + 'ListAction',
    method: 'post',
    data
  })
}

export function CancelAction(data) {
  return request({
    url: api + 'CancelAction',
    method: 'post',
    data
  })
}


